src/task1/task1.js

```diff
@@ -0,0 +1,9 @@
+function func(n){
+    let result = "";
+    for (let i = '#' ; i.length<n; i+='#'){
+         result+=i;
+        result+='\n';
```

Comment:
4-5 lines - Could be one line action, isn't it?

Reply:
fair enough